src/components/adjudications/AdjudicationDialog.jsx

```diff
@@ -44,25 +44,38 @@ class AdjudicationDialog extends React.Component {
       technicalMark
     } = currentValues;
 
+    const {
+      academicLevel,
+      choreographers,
+      danceEntry,
+      danceStyle,
+      danceTitle,
+      performers,
+      school,
+      size
+    } = performanceValues;
+
+    console.log(performanceValues);
+
     const viewForm = (
       <React.Fragment>
         <div style={{ display: 'flex', flexFlow: 'column', margin: '35px' }}>
           <div style={{ display: 'flex', justifyContent: 'space-around', borderBottom: '1px solid #dcdcdc' }}>
             <div style={{ flex: '1 0 0' }}>
-              <DialogReadOnly defaultValue="hi" label="Dance Entry" />
-              <DialogReadOnly defaultValue="hi" label="Dance Title" />
-              <DialogReadOnly defaultValue="hi" label="Performers" />
-              <DialogReadOnly defaultValue="hi" label="Choreographer" />
+              <DialogReadOnly defaultValue={danceEntry ? danceEntry : "N/A"} label="Dance Entry" />
+              <DialogReadOnly defaultValue={danceTitle ? danceTitle : "N/A"} label="Dance Title" />
+              <DialogReadOnly defaultValue={performers ? performers : "N/A"} label="Performers" />
+              <DialogReadOnly defaultValue={choreographers ? choreographers : "N/A"} label="Choreographer" />
             </div>
 
             <div style={{ flex: '1 0 0' }}>
               <div style={{ display: 'flex' }}>
-                <DialogReadOnly label="School" />
-                <DialogReadOnly label="Level" />
+                <DialogReadOnly defaultValue={school ? school : "N/A"} label="School" />
```

Comment:
do we want this to be "defaultValue" or  is there a value parameter?

The use of school ? school : "N/A" means if the first exists, let's take that value, else "N/A". This is logically equivalent to the Elvis operator from Java `A ?: B` which takes A if it is truthy, else B. The javascript equivalent is `A || B`. It will take A if it exists, else it will return B.

Ref: https://stackoverflow.com/questions/6613952/is-there-a-null-coalescing-elvis-operator-or-safe-navigation-operator-in-javas

Also, All of these are the exact same check, maybe we should add an inline function to handle this?

```
const handleInvalidData = (str) => str || "N/A";
...
defaultValue ={handleInvalidData(school / performers / etc.)}
```